python/pyspark/sql/dataframe.py

```diff
@@ -1393,6 +1393,16 @@ def replace(self, to_replace, value=None, subset=None):
         |null|  null| null|
         +----+------+-----+
 
+        >>> df4.na.replace('Alice', None).show()
```

Comment:
I've not noticed that. Why we test `dataframe.na.replace` at the doc test of `dataframe.replace`? We should test `dataframe.replace` here.